Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:8
+
+RUN apt-get update && apt-get install -yq gconf-service libasound2 libatk1.0-0 libc6 libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget && rm -r /var/lib/apt/lists/*
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json .
+
+RUN npm install
+
+# Bundle app source
+COPY . .
```

Comment:
Is there a reason we add another layer to the image? We could save on this additional layer by including this when we include package.json

Reply:
The reason is to be able to cache `package.json` and run `npm install` only when `package.json` changes. If we bundle everything in one layer, a change to any source file would lead to a new `npm install`.

Reply:
@strattadb makes sense